portingdb/load_data.py

```diff
@@ -15,6 +15,10 @@
 PY2_STATUSES = {'released', 'legacy-leaf', 'py3-only'}
 DONE_STATUSES = PY2_STATUSES | {'dropped'}
 
+# Current Fedora version, used to get info about long-term FTBFSs
+# Update to the Rawhide version after a mass rebuild.
+CURRENT_FEDORA = 29
```

Comment:
We might advice to do it a month or two after a mass rebuild not to have the portingdb overwhelmed by this.

Reply:
Just use common sense when updating it :)